### ISSUE TITLE
[Slack] Can't add reactions/emojis

### ISSUE THREAD

**Issue by RaydenX93** [TheGoddessInari/rambox#42]
### Steps to reproduce

1. Try to add a reaction/emoji to a post in a Slack channel.

### Expected behavior

The reaction/emoji should appear in the post.

### Actual behavior

Nothing happens.

### ENVIRONMENT

> Rambox-OS: v0.5.18-771266f

> OS: Win 10

> Arch: x64

**Comment by TheGoddessInari** [comment]
You can type the emoji in that it would've put in, such as `:hushed:` and it'll work, it's just not pasting what you're selecting.

I'm attempting to drill down into why this isn't working, but live debugging of a website isn't my forte. Any help, even if debugging to figure out why this isn't working, is appreciated.

Looks like there may be problems with how the per-view devtools are setup, as well. Breaking freezes the devtools as well currently. I'd like to meet the person who came up with that. ❤️

**Comment by TheGoddessInari** [comment]
fwiw, I still can't debug this on Rambox, even with Electron 3.0.8. It doesn't want to generate an exception anymore, so even pausing on caught exceptions is ineffective.

If someone could help me find the actual function(s) responsible for inserting the emoji, at least I could try to make breakpoints and figure out what's going wrong. Do to obfuscation, this'd have to be a snippet of code along with line number in the Pretty Printed javascript, as I'm pretty sure there aren't exactly any unique function names left in the source.

The source is so obfuscated, I haven't been able to locate the relevant part where it inserts the emoji into the text input area.

It no longer appears to be triggering any exception as far as Rambox-OS can tell, but for all I know, they've simply stepped up their anti-debugging techniques.

**Comment by TheGoddessInari** [comment]
I'm still not sure what's going on here. THey used to catch and do something with the exceptions caused by this. Now they're avoiding exceptions even as it doesn't work.

The code is so obfuscated, I'm not even sure how this would get traced.

I don't suppose anyone has any ideas? It's likely to be Electron-specific. As far as I could tell, plain Chromium didn't have this issue.

**Comment by JakobStruye** [comment]
I'm trying to look into this but I can't seem to figure out how to get the dev tools for a specific service. 
Judging by 4b24171214612d515da634a17452b74b51ed6c09 the Toggle Dev Tools button should open the dev tools for the service but I'm only seeing the global one. Any help on that?

I also tried recreating the issue in other electron-based apps but there it did work as intended.

**Comment by TheGoddessInari** [comment]
Right click on the tab and `Toggle Developer Tools` there.

**Comment by JakobStruye** [comment]
I think I found where it goes wrong. I know nothing about js so apologies if my explanation is hard to follow. Anyway, here goes:

 The name of the emoji ends up undefined in Rambox.
It's all file up until this point:
![image](https://user-images.githubusercontent.com/3791759/52114541-98691f80-260c-11e9-8a69-ffd244736a82.png)
(selected step in the call stack is the one _below_ window.setTimeout, not the one with the blue arrow)
Here n.name is still the emoji's name (which is later on used to send to the API)

The setTimeout called appears to be this one:
![image](https://user-images.githubusercontent.com/3791759/52114664-ebdb6d80-260c-11e9-9098-f3eacba1b23c.png)
Which seems to have only 2 params (while the name was the third)

Just past that the code ends up here
![image](https://user-images.githubusercontent.com/3791759/52114743-20e7c000-260d-11e9-8abe-35cf804563ef.png)
Note that now n is undefined, while I confirmed that in Chrome n is at that point still the emoji name.


In Chrome the call stack looks a bit different around that point:
![image](https://user-images.githubusercontent.com/3791759/52114818-57bdd600-260d-11e9-911f-3f7c9fa42223.png)
Mainly note that there's no separate entry for window.setTimeout. This leads me to believe a different implementation of it is called in Chrome, if that makes any sense at all.
